core/domain/user_services.py

```diff
@@ -41,14 +41,68 @@
 
 
 class UserSettings(object):
-    """Value object representing a user's settings."""
+    """Value object representing a user's settings.
+
+    Attributes:
+        user_id: str. The user id.
+        email: str. The user email.
+        username: str or None. Identifiable username to display in the UI.
+        last_agreed_to_terms: datetime.datetime or None. When the user last
+            agreed to the terms of the site.
+        last_started_state_editor_tutorial: datetime.datetime or None. When
+            the user last started the state editor tutorial.
+        last_logged_in: datetime.datetime or None. When the user last logged in.
+        last_created_an_exploration: datetime.datetime or None. When the user
+            last created an exploration.
+        last_edited_an_exploration: datetime.datetime or None. When the user
+            last edited an exploration.
+        profile_picture_data_url: str or None. User uploaded profile picture as
+            a dataURI string.
+        user_bio: str or None. User specified biography.
+        subject_interests: str or None. Subject interests specified by the user.
```

Comment:
It is a unicode string know ? I inferred its type from `UserSettingsModel` in `gae_models.py` (L55).

```
subject_interests = ndb.StringProperty(repeated=True, indexed=True)
```

[Python app engine](https://cloud.google.com/appengine/docs/python/ndb/entity-property-reference#Python_Properties_and_value_types) specifies its python type as `str unicode`. What should I ideally put in docstring (`str or None`/`unicode str or None`/`ndb.StringProperty`)?